server/util/body-parser.js

```diff
@@ -341,7 +341,7 @@ function getImageFromWpNode(node) {
     node.attrs && getAttrVal(node.attrs, 'class') === 'wp-caption-text'
   );
   const alt = getAttrVal(img.attrs, 'alt');
-  const description = getAttrVal(img.attrs, 'data-image-description');
+  const copyright = getAttrVal(img.attrs, 'data-image-description');
```

Comment:
Hmm. I do wonder if this is the best place for it.
There is a lot of images already in the lib with descriptions, and these will show up.

We could have something along the lines of `getCopyright` that does a check for specific data.

Or we talk to @jtweed's team and see where else we might store this info (but know that we need this soon).